app/controllers/mentors_controller.rb

```diff
@@ -1,5 +1,6 @@
 class MentorsController < ApplicationController
   def index
+    mentor = Mentor.where(profile_complete: true)
   end
```

Comment:
@bfpepper We are actually not passing anything through this controller, it's all done with ajax, so this won't work. The rest of these changes make sense, but we have to do the filtering on the component and can't pass it in from here. 
@jbkimble